index.html

```diff
@@ -4,133 +4,143 @@
-            <li class="main-nav__item">HTML Academy</li>
+            <li class="main-nav__item main-nav__item--active"><a class="main-nav__anchor" href="index.html">Home</a></li>
+            <li class="main-nav__item"><a class="main-nav__anchor" href="photos.html">Photos and videos</a></li>
+            <li class="main-nav__item"><a class="main-nav__anchor" href="form.html">Feedback Form</a></li>
+            <li class="main-nav__item"><a class="main-nav__anchor" href="#">HTML Academy</a></li>
           </ul>
-          <button class="main-nav__toggle main-nav__toggle--close"></button>
-          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+          <div class="main-nav__toggle main-nav__toggle--close">
+            <div class="main-nav__menu-bar main-nav__menu-bar--top"></div>
+            <div class="main-nav__menu-bar main-nav__menu-bar--middle"></div>
+            <div class="main-nav__menu-bar main-nav__menu-bar--bottom"></div>
+          </div>
+          <a class="main-nav__link" href="index.html"><img src="images/logo.png" class="main-nav__logo" alt="Sedona logo"></a>
         </nav>
 
-        <div calss="intro">
-          <ul class="intro-list">
-            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
-            <li class="intro-list__item intro-list__item--middle">Sedona</li>
-            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
-          </ul>
+        <div class="intro">
+          <div class="intro-images"></div>
         </div>
+        <div class="header-separator"></div>
       </header>
 
+
       <main class="main-content">
         <section class="preamble">
           <div class="preamble__inner">
-            <h2 class="preamble__header">SEDONA - a small town in the ARIZONA, deserves more!</h2>
+            <h2 class="preamble__header">SEDONA - a small town</br>in the ARIZONA,</br>deserves more!</h2>
             <p class="preamble__paragraph">Consider 5 reasons why Sedona is cooler than Grand Canyon!</p>
           </div>
+          <div class="preamble-separator"></div>
         </section>
 
         <section class="sightseeing">
           <div class="sightseeing__inner">
             <h2 class="sightseeing__header">The Present Town</h2>
             <p class="sightseeing__counter-paragraph">— №1 —</p>
             <p class="sightseeing__paragraph">Sedona is not a tourist attraction, the life flows in here in its own way.</p>
-            <img class="sightseeing__image" src="" alt="Building in the city"/>
           </div>
         </section>
 
-      <section class="recommendations">
-        <div class="recommendations__inner">
-          <ul class="recommendations__list">
-            <li class="recommendations__item">
-              <!--house-pic-->
+        <div class="image-wrapper image-wrapper--top"></div>
```

Comment:
Image replacement, put text incide https://css-tricks.com/the-image-replacement-museum/